spec/controllers/proposals_controller_spec.rb

```diff
@@ -91,16 +91,22 @@
       end
     end
 
-    context 'details beta' do
+  end
+
+  describe '#show + new details' do
+    before do
```

Comment:
@jessieay Got it. I think this particular `before` block is valid because its the flag to trigger the cookie. Without the cookie, none of the following specs will be properly working. 

When we remove the cookie flag, it will be easy to remove the content in the before block.

What do you think?